README.md

```
@@ -1,231 +1,463 @@
-Tools for working with the APIs of Zotero and Zenodo (zotzen)
+## Table of Contents
 
-This repository is part of a set of repositories, [see here](https://github.com/orgs/OpenDevEd/teams/zotzen-team/repositories). Currently, this set contains a number of libraries
+- [Installation](#installation)
+- [Quick Start](#quick-start)
+- [CLI Commands](#cli-commands)
+- [Library Usage](#library-usage)
+- [Configuration](#configuration)
+- [Development](#development)
+- [API Reference](#api-reference)
+- [Contributing](#contributing)
```

markdownlint-cli2 (0.20.0)

[warning] Link fragments should be valid

(MD051, link-fragments)